python/ray/util/state/state_cli.py

```diff
@@ -81,7 +81,12 @@ def _parse_filter(filter: str) -> Tuple[str, PredicateType, SupportedFilterType]
         filter[predicate_index[0] : predicate_index[1]],
         filter[predicate_index[1] :],
     )
-
+    if not key.isalpha():
+        raise ValueError(
+            f"The format of a given filter {filter} is invalid: "
+            f"key {key} is not an alphabetic string. "
+            "Please provide key=val or key!=value"
+        )
```

Comment:
i removed a value.isalnum check because there's tests which have it with symbols such as key=value=123, not sure if having symbols in the value is ever a valid thing